frontend/tests/common/queries/filter.test.ts

```diff
@@ -13,9 +14,11 @@ import {
 } from "src/common/queries/filter";
 import { PUBLICATION_DATE_VALUES } from "src/components/common/Filter/common/constants";
 
+const { describe } = test;
+
 describe("filter", () => {
   describe("Calculate Months Since Publication", () => {
-    it("calculates one month since publication", () => {
+    test("calculates one month since publication", () => {
```

Comment:
playwright test runner doesn't have global variables available, so we need to destruct from `test` namespace 
also no `it`, just `test`